packages/relayer/processor/process_message.go

```diff
@@ -243,11 +245,11 @@ func (p *Processor) sendProcessMessageCall(
 		auth.GasLimit = 3000000
 	} else {
 		// otherwise we can estimate gas
-		gas, cost, err = p.estimateGas(ctx, event.Message, proof)
+		gas, err = p.estimateGas(ctx, event.Message, proof)
 		// and if gas estimation failed, we just try to hardcore a value no matter what type of event,
 		// or whether the contract is deployed.
 		if err != nil || gas == 0 {
```

Comment:
shall we print this `err` in logs here? or its safe to ignore?

Reply:
if its not nil then hardcode so safe to ignore?

Reply:
adding slog.Info for debugging purposes